media/windows/ECC-MQTT-IoT/ECC_MQTT_IoT_SQLite.py

```diff
@@ -101,16 +124,149 @@
     print(F"ECC MQTT IoT Listener application, version {eccmqtt_iot_version}, {eccmqtt_iot_date}...")
     sys.exit(0)
 
+
+class GZipRotator(logging.handlers.RotatingFileHandler):
+
+    def doRollover(self):
+
+        global done_flag
+
+        # Let the default log rollover routine do its thing first...
+        super().doRollover()
+
+        # Construct a file pattern to match all current gzip'd logs
+        base_file_pattern = self.baseFilename + ".[0-9].gz"
```

Comment:
```suggestion
        base_file_pattern = self.baseFilename + ".[0-9]+.gz"
```

I think you want the `+` in there so that you also catch files like `foo.10.gz`.

Reply:
True, if we want to make this a more generic routine.  The code currently has a default of 9 archives, but since it _could_ be changed to a higher value, this is better...